### PR TITLE
Added moondream and llava.

### PR DESCRIPTION
**Pull request by MaanavD**

### CODE REVIEW

**Comment by MaanavD** [comment]
All accuracy and tests are passing on A100 for both llava and moondream. 
Please approve workflows when able @xuzhao9 :)

File: torchbenchmark/util/framework/huggingface/patch_hf.py [hunk 1]
```diff
@@ -5,16 +5,22 @@
...
 
 PATCH_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "patches")
 
 def cache_model(name: str, **kwargs):
     import transformers
     model_config = eval(class_models[name][2])
     model_ctor = getattr(transformers, class_models[name][3])
-    model_ctor.from_config(model_config, **kwargs)
+    print(model_config.__class__.__name__)
+    print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
```
Comment (xuzhao9) [inline torchbenchmark/util/framework/huggingface/patch_hf.py:18]: Are these prints for debugging purpose?

File: torchbenchmark/util/framework/huggingface/patch_hf.py [hunk 2]
```diff
@@ -5,16 +5,22 @@
...
+    if (model_config.__class__.__name__ is "PhiConfig" or "LlavaConfig"):
```
Comment (xuzhao9) [inline torchbenchmark/util/framework/huggingface/patch_hf.py:19]: I am wondering why we can't use the same code from `model_factory.py` to determine whether `from_config` should be used?

**Review by xuzhao9 — COMMENTED** [review]
See inline comments

File: torchbenchmark/util/framework/huggingface/patch_hf.py (hunk 2 above)
Comment (MaanavD) [reply torchbenchmark/util/framework/huggingface/patch_hf.py:17]: Causes errors w/ runs for other HF models (feel free to test & replicate)

**Comment by xuzhao9** [comment]
The models have been added by https://github.com/pytorch/benchmark/pull/2176

**Pull request closed without merging**